pkg/cnbutils/buildpack.go

```diff
@@ -77,32 +103,32 @@ func DownloadBuildpacks(path string, bpacks []string, dockerCreds string, utils
 func copyBuildPack(src, dst string, utils BuildUtils) error {
 	buildpacks, err := utils.Glob(filepath.Join(src, "*"))
 	if err != nil {
-		return fmt.Errorf("failed to read directory: %s, error: %s", src, err.Error())
+		return errors.Wrapf(err, "failed to read directory: %s", src)
 	}
 
 	for _, buildpack := range buildpacks {
 		versions, err := utils.Glob(filepath.Join(buildpack, "*"))
 		if err != nil {
-			return fmt.Errorf("failed to read directory: %s, error: %s", buildpack, err.Error())
+			return errors.Wrapf(err, "failed to read directory: %s", buildpack)
 		}
 		for _, srcVersionPath := range versions {
 			destVersionPath := filepath.Join(dst, strings.ReplaceAll(srcVersionPath, src, ""))
 
 			exists, err := utils.FileExists(destVersionPath)
 			if err != nil {
-				return fmt.Errorf("failed to check if directory exists: '%s', error: '%s'", destVersionPath, err.Error())
+				return errors.Wrapf(err, "failed to check if directory exists: '%s'", destVersionPath)
 			}
 			if exists {
 				utils.RemoveAll(destVersionPath)
 			}
 
 			if err := utils.MkdirAll(filepath.Dir(destVersionPath), 0755); err != nil {
-				return fmt.Errorf("failed to create directory: '%s', error: '%s'", filepath.Dir(destVersionPath), err.Error())
+				return errors.Wrapf(err, "failed to create directory: '%s'", filepath.Dir(destVersionPath))
 			}
 
 			err = utils.FileRename(srcVersionPath, destVersionPath)
```

Comment:
We are moving it out of the cache folder, this is wrong, right? We should copy it. I thought we changed that.